media/css/firefox/desktop/trust.less

```diff
@@ -48,24 +48,38 @@
         .font-size(18px);
     }
 
-    .feature-list li {
-        background: url(/media/img/firefox/desktop/trust/sprite-privacy.png) center top no-repeat;
-        padding-top: 180px;
-
-        &#privacy-dnt {
-            background-position: 70px top;
-        }
-
-        &#privacy-browsing {
-            background-position: -390px top;
-        }
-
-        &#privacy-lightbeam {
-            background-position: -850px top;
+    .feature-list {
+        h3 {
+            line-height: 1.3;
         }
-
-        &#privacy-forget {
-            background-position: -1310px top;
+        li {
+            float: none;
+            display: inline-block;
+            background: url(/media/img/firefox/desktop/trust/sprite-privacy.png) center top no-repeat;
+            margin-right: 30px;
```

Comment:
Is it intentional that this has a right margin of `30px`, while the `last-child` below has a left margin of `20px`?